tests/neg-custom-args/fatal-warnings/i15503b.scala

```diff
@@ -0,0 +1,49 @@
+// scalac: -Wunused:locals
```

Comment:
I think it would be nice to add a test case that has all flags enabled, and tests them together in the same trees.

Reply:
I added tests for `-Wunused:linted` and `-Wunused:all`
Thank you for the review